src/shogun/mathematics/linalg/LinalgNamespace.h

```diff
@@ -1325,6 +1325,39 @@ namespace shogun
 			return result;
 		}
 
+		/** Performs the operation B = cos(A)
```

Comment:
noooooo
You would cherry pick your other commit instead of force pushing this into here. This just wastes CI cycles.
git is a great tool and it can avoid exactly those situations

Reply:
now (with the other PR merged), you need to force push here again. If you had cherry picked the commit, this wouldnt have been necessary (git would have worked it out for us)
next time :)

Reply:
this still needs to be removed